hub/integrations/common/utils.py

```diff
@@ -0,0 +1,16 @@
+from hub.core.dataset import Dataset
```

Comment:
Not sure if this location for the shared utils would work?

Reply:
should be fine